internal/util/secretskms.go

```diff
@@ -38,6 +40,9 @@ const (
 	// kmsTypeSecretsMetadata is the SecretsKMS with per-volume encryption,
 	// where the DEK is stored in the metadata of the volume itself.
 	kmsTypeSecretsMetadata = "metadata"
+
+	metadataSecretNameKey      = "secretName"
```

Comment:
It would be good to document these. You commented the behaviour inside the function where these are used, but commenting them here is very helpful for people if they do not want to read through the code and find out.